src/org/labkey/test/BootstrapLocators.java

```diff
@@ -2,8 +2,13 @@
 
 public abstract class BootstrapLocators
 {
-    static public Locator infoAlert = Locator.tagWithClass("div", "alert-info");
-    static public Locator successAlert = Locator.tagWithClass("div", "alert-success");
-    static public Locator dangerAlert = Locator.tagWithClass("div", "alert-danger");
-    static public Locator warningAlert = Locator.tagWithClass("div", "alert-warning");
+    public static final Locator infoAlert = Locator.tagWithClass("div", "alert-info");
+    public static final Locator successAlert = Locator.tagWithClass("div", "alert-success");
+    public static final Locator dangerAlert = Locator.tagWithClass("div", "alert-danger");
+    public static final Locator warningAlert = Locator.tagWithClass("div", "alert-warning");
+    
```

Comment:
I've changed these in my branch to be *Banner and not *Alert, but I think you have already seen that.
It will just get resolved during merge.